database/impl/schemas/app.aaps.database.impl.AppDatabase/22.json

```diff
@@ -0,0 +1,3605 @@
+{
```

Comment:
This file is not necessary

Reply:
HeartRateDaoTest tests the creation of the heartrate table, i.e. the DB migration from v 22. For that the file is needed.